contrib/librespot.service

```diff
@@ -9,7 +9,7 @@ User=nobody
 Group=audio
 Restart=always
 RestartSec=10
-ExecStart=/usr/bin/librespot -n "%p on %H"
+ExecStart=/bin/sh -c 'exec /usr/bin/librespot --name librespot@$(hostname -s)'
```

Comment:
Do you really think it's necessary to spawn another shell here around for librespot? I understand the problem, and maybe I'm a little OCD here, but I think the source of the bug should be fixed, which is that librespost should probably encode the device name so it's safe for use in a DNS node.
![screen shot 2017-06-15 at 12 25 43](https://user-images.githubusercontent.com/20795/27177682-774ccd58-51c7-11e7-93a9-8a377209d383.png)
 I've tried to give the iOS Spotify app a device name with a point in it, and there it works.